### DISCUSSION TITLE
First documentation contribution

### DISCUSSION THREAD

**Discussion by abeldebruijn** [supabase/supabase#1809] [Questions] [answered]
Watching the Build in public youtube series from Supabase inspired me to write a quick start guide for Svelte. So I pulled the master branch and ran `npm install` in the web directory. However when I try to run `npm run start` I get these error messages in my terminal.

![Screenshot 2021-06-03 at 14 42 27](https://user-images.githubusercontent.com/25432790/120646543-eced6580-c479-11eb-856a-3f48a48a7290.png)

I am not that experienced with Docusaurus and getting it started. Could anybody nudge me in the right direction of resolving my issue?

**Comment by inian** [comment, marked as answer]
Hey @yustarandomname, check out [this](https://github.com/supabase/supabase/blob/master/development.md) for getting started on development. 
You need to do a 
```
cd web/spec
make
npm run gen:supabase
npm run build
npm run start
```
